__tests__/topics.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const testData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+const db = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("/api/topics", () => {
+  test.only("Get 200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        //console.log(body);
+        const { topics } = body;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
```

Comment:
Nice use of forEach✅
`.toMatchObject` or `.objectContaining` could be good matchers to use here